_example/chi/main.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"log"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/arl/statsviz"
+	"github.com/go-chi/chi"
+)
+
+func main() {
+
+	// Force the GC to work to make the plots "move".
+	go work()
```

Comment:
To keep all examples as small and uniform as possible, there's a `work` function in the `_example` package so you don't need to copy it here.

Can you check other examples out and import `work` please?
https://github.com/arl/statsviz/blob/5ea9f0cf1bd172b759c0b7b8bd2f790275ac3ec4/_example/root/main.go#L13

`example` is imported as:
https://github.com/arl/statsviz/blob/5ea9f0cf1bd172b759c0b7b8bd2f790275ac3ec4/_example/root/main.go#L3-L9

Reply:
Makes sense! I did not see this in the other examples.